src/components/footer-navigation/index.tsx

```diff
@@ -1,27 +1,50 @@
-import { FNB_TRAVELLER } from '@/constants/NAVIGATE_BUTTON'
+import {
+  FNB_GUEST,
+  FNB_TRAVELLER,
+  FNB_TRAVELLY,
+  IMenu,
+} from '@/constants/NAVIGATE_BUTTON'
+import { RootState } from '@/store/store'
+import { LoginUserRoleType } from '@/types/userRole.type'
 
 import NavigateButton from '@components/navigate-button'
+import { useSelector } from 'react-redux'
 import { useLocation } from 'react-router-dom'
 
 import * as S from './FooterNavigation.style'
 
-//TODO: 회원에 따라(판매자, 구매자) FNB가 다르게 보이는 기능
 //TODO: 임시 키
 
+const MENU_MAP: Record<LoginUserRoleType, IMenu[]> = {
+  traveller: FNB_TRAVELLER,
+  travelly: FNB_TRAVELLY,
+}
+
 function FooterNavigation() {
   const location = useLocation()
+  const { role } = useSelector((state: RootState) => state.auth)
 
   return (
     <S.Container>
-      {FNB_TRAVELLER.map(({ img, description, url }) => (
-        <NavigateButton
-          key={img}
-          img={img}
-          description={description}
-          url={url}
-          isButtonActive={location.pathname === url}
-        />
-      ))}
+      {role
+        ? MENU_MAP[role].map(({ img, description, url }) => (
+            <NavigateButton
+              key={img}
+              img={img}
+              description={description}
+              url={url}
+              isButtonActive={location.pathname === url}
+            />
+          ))
+        : FNB_GUEST.map(({ img, description, url }) => (
+            <NavigateButton
+              key={img}
+              img={img}
+              description={description}
+              url={url}
+              isButtonActive={location.pathname === url}
+            />
+          ))}
```

Comment:
P3: 상단에서 로직을 분리하면 좀 더 간결하게 처리할 수 있을 것 같습니다!
```suggestion
const footer = role ? MENU_MAP[role] : FNB_GUEST;

  return (
    <S.Container>
      {footer.map(({ img, description, url }) => (
        <NavigateButton
          key={img}
          img={img}
          description={description}
          url={url}
          isButtonActive={location.pathname === url}
        />
      ))}
    </S.Container>
  );
}

```